scripts/gz10/gz10.py

```diff
@@ -93,9 +93,7 @@ def _info(self):
         features = datasets.Features(
             {
                 "gz10_label": datasets.Value("int32"),
-                "ra": datasets.Value("float32"),
-                "dec": datasets.Value("float32"),
```

Comment:
Why are we dropping these keys? I think they should be included for cross matching.